R/create_hexsticker.R

```diff
@@ -0,0 +1,29 @@
+#' @title Make hexagonal logo for package
+#'
+#' @description This function makes a hexsticker in INBO style for the provided
+#' package name.
+#'
+#' @param package_name package name that should be mentioned on the hexsticker
+#' @param filename filename to save the sticker
```

Comment:
maybe place the logo by default in the `man/figures` folder. That's where pkgdown needs it.

Reply:
done